src/Table.cpp

```diff
@@ -101,4 +101,37 @@ namespace voltdb {
             ostream << std::endl;
         }
     }
+
+    void Table::operator >> (std::ostream &ostream) const {
+
+        int32_t size = m_buffer.limit();
+        ostream.write((const char*)&size, sizeof(size));
+        if (size != 0) {
+            ostream.write(m_buffer.bytes(), size);
+        }
+    }
+
+    bool Table::operator!=(const Table& rhs) const {
+        // ToDo: Implement comparison ... byte comparison of buffer???
+        return m_buffer != rhs.m_buffer;
```

Comment:
I will change and implement == for table that account for row and column. Ideally Column should also have equality. I will implement that too...you are right lets get it right.